File: Bubble_Sort.js
'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', function(inputStdin) {
    inputString += inputStdin;
});

process.stdin.on('end', function() {
    inputString = inputString.split('\n');

    main();
});

function readLine() {
    return inputString[currentLine++];
}

/*
 * Complete the 'countSwaps' function below.
 *
 * The function accepts INTEGER_ARRAY a as parameter.
 */

function countSwaps(a) {
    // Write your code here
let temp,counter =0;
 for(let i=0;i<a.length ;i++){
        for(let j=i;j<a.length ;j++){
            if(a[j]<a[i]){
                temp =a[i];
                a[i]=a[j];
                a[j]=temp;
                counter++;
            }
        }
    }
    
    console.log("Array is sorted in %d swaps.",counter);
    console.log("First Element: %d",a[0]);
    console.log("Last Element: %d",a[a.length-1]);
}

function main() {
    const n = parseInt(readLine().trim(), 10);

    const a = readLine().replace(/\s+$/g, '').split(' ').map(aTemp => parseInt(aTemp, 10));

    countSwaps(a);
}

File: Bubble_Sort.py
#!/bin/python3

import math
import os
import random
import re
import sys

#
# Complete the 'countSwaps' function below.
#
# The function accepts INTEGER_ARRAY a as parameter.
#

def countSwaps(a):
    # Write your code here
    counter  =0
    for i in range(len(a)):
      for j in range(i,len(a)):
        if a[j]<a[i]:
          a[j],a[i] =a[i],a[j]
          counter+=1
          
    print("Array is sorted in "+ str(counter)+" swaps.")
    print("First Element:",a[0])
    print("Last Element:",a[-1])
          

if __name__ == '__main__':
    n = int(input().strip())

    a = list(map(int, input().rstrip().split()))

    countSwaps(a)
